.github/workflows/e2e_tests.yml

```diff
@@ -0,0 +1,211 @@
+
+      - name: pilot-tests
+        run: |
+          go test -tags=integ -timeout 30m -run='TestEmptyCluster|TestFileOnly|TestDirectoryWithoutRecursion|TestDirectoryWithRecursion|TestInvalidFileError|TestJsonInputFile|TestJsonOutput|TestKubeOnly|TestFileAndKubeCombined|TestAllNamespaces|TestTimeout|TestErrorLine|TestWait|TestVersion|TestDescribe|TestAddToAndRemoveFromMesh|TestProxyConfig|TestProxyStatus|TestAuthZCheck|TestLocality|TestMain|TestMirroring|TestMirroringExternalService|TestTproxy|TestValidation|TestEnsureNoMissingCRDs|TestWebhook' ./tests/integration/pilot/ -istio.test.skipVM true -p 1 -test.v 
+          go test -tags=integ ./tests/integration/pilot/analysis/... -istio.test.skipVM true -p 1 -test.v
+          go test -tags=integ ./tests/integration/pilot/revisions/... -istio.test.skipVM true -p 1 -test.v
+      
+      - name: telemetry-tests
+        run: go test -tags=integ -timeout 30m -run='TestStatsFilter|TestStatsTCPFilter|TestSetup|TestIstioctlMetrics|TestTcpMetric|TestStatsFilter|TestWASMTcpMetric|TestWasmStatsFilter|TestMain|TestCustomizeMetrics'  ./tests/integration/telemetry/stats/... -istio.test.skipVM true -p 1 -test.v
+
+      - name: security-tests
+        run: |
+          go test -tags=integ  ./tests/integration/security/ca_custom_root/... -istio.test.skipVM true -p 1 -test.v
+          go test -tags=integ  ./tests/integration/security/file_mounted_certs/... -istio.test.skipVM true -p 1 -test.v
+          go test -tags=integ  ./tests/integration/security/filebased_tls_origination/... -istio.test.skipVM true -p 1 -test.v
+          go test -tags=integ  ./tests/integration/security/mtls_first_party_jwt/... -istio.test.skipVM true -p 1 -test.v
+          go test -tags=integ  ./tests/integration/security/mtlsk8sca/... -istio.test.skipVM true -p 1 -test.v
+          go test -tags=integ  ./tests/integration/security/sds_egress/... -istio.test.skipVM true -p 1 -test.v
+          go test -tags=integ  ./tests/integration/security/sds_tls_origination/... -istio.test.skipVM true -p 1 -test.v
+          go test -tags=integ  ./tests/integration/security/webhook/... -istio.test.skipVM true -p 1 -test.v
+
+      - name: aks-cluster-delete
+        run: bash ./tetrateci/cleanup_aks_cluster.sh
+
+  make_release:
+    strategy:
+      matrix:
+        os: [ ubuntu-latest ]
+    
+    needs: [eks-e2e-test, gke-e2e-test, aks-e2e-test]
+
+    name: release-builder-run
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+
+      - name: Login to Bintray
+        uses: docker/login-action@v1
+        with:
+          registry: hellozee-docker-istio-temp.bintray.io
+          username: ${{ secrets.BINTRAY_USER }}
+          password: ${{ secrets.BINTRAY_API_KEY }}
+      - name: Get tag
+        id: tag
+        uses: dawidd6/action-get-tag@v
+
+      - name: run release builder
+        env:
+          TAG: 1.8.0-test #need to replace this with ${{ steps.tag.outputs.tag }}
```

Comment:
Need to set HUB to bintray docker registry

Reply:
Ahh yes, there are some hardcoded stuff I used for trying those out, once I have the replacements.